### PR TITLE
Add pulumi community

### PR DESCRIPTION
**Pull request by simenandre**

### CODE REVIEW

File: communities/pulumi.yml [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+    - https://github.com/pulumi/test-repo
+    - https://github.com/pulumi/frontend-working-group
+    - https://github.com/pulumi/blog-lambda-snapshots
+    - https://github.com/pulumi/pulumi-upgrade-provider-action
+    - https://github.com/pulumi/pulumi-hubspot
+    - https://github.com/pulumi/orbit-actions
+    - https://github.com/pulumi/pulumi-nodejs-dynamic
+  governance-repositories:
+    repo_urls:
+    - https://github.com/pulumi/pulumi
```
Comment (eyehwan) [inline communities/pulumi.yml:187]: Hi, @simenandre  thanks for your PR:

Normally, repositories in the two categories should not overlap. The former is used to store software artifacts source code, which are software components that accompany version releases, while the latter is used to store code repositories related to community governance, such as COCs, contributor guides, charters, etc.

Does this code repository at https://github.com/pulumi/pulumi have two classifications of attributes simultaneously?

BR//Yehui.

Comment (simenandre) [reply communities/pulumi.yml:187]: Yes, I haven't found any other repository where governance is done, so I guess it has two. We could also remove from governance?

Comment (eyehwan) [reply communities/pulumi.yml:187]: Yes, you can remove it from governance type which is optional, otherwise data would be duplicated in general.

Comment (simenandre) [reply communities/pulumi.yml:187]:
```suggestion
```

Like this, or should we remove `governance-repositories` as well?

File: communities/pulumi.yml [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+    - https://github.com/pulumi/pulumi-synced-folder
+    - https://github.com/pulumi/test-repo
+    - https://github.com/pulumi/frontend-working-group
+    - https://github.com/pulumi/blog-lambda-snapshots
+    - https://github.com/pulumi/pulumi-upgrade-provider-action
+    - https://github.com/pulumi/pulumi-hubspot
+    - https://github.com/pulumi/orbit-actions
+    - https://github.com/pulumi/pulumi-nodejs-dynamic
+  governance-repositories:
+    repo_urls:
```
Comment (eyehwan) [inline communities/pulumi.yml:186]:
```suggestion
    repo_urls: []
```

**Comment by eyehwan** [comment]
with an empty list :）

**Comment by eyehwan** [comment]
@simenandre Hi Simen, report is ready: https://oss-compass.org/analyze?label=pulumi&level=community&range=5Y

One thing may need your attention is that some of your repos haven't updated for a long time, that makes "[updated since](https://oss-compass.org/analyze?label=pulumi&level=community&range=5Y#activity_updated_since) " metric result in Activity model is not that good. 
The similar situation is also feedback by " Is Maintained" metric in Collaboration Development Index model .

**Pull request merged**